packages/api-plugin-fulfillment/src/simpleSchemas.js

```diff
@@ -0,0 +1,133 @@
+import SimpleSchema from "simpl-schema";
+
+/**
+ * @name MethodEmptyData
+ * @memberof Schemas
+ * @type {SimpleSchema}
+ * @summary Defines Empty placeholder
+ * @property {String} gqlType Defines the method type
+ * @property {Boolean} emptyData Empty Data fields
+ */
```

Comment:
Why do we have a JSDoc here and then for no other Schemas?

Reply:
Added JSDoc for the remaining Schemas as well